config/config.sample.pythtest.toml

```diff
@@ -60,3 +60,12 @@ exporter.compute_unit_price_micro_lamports = 1000
 # Note that this doesn't affect the rate at which transactions are published:
```

Comment:
let's hide it from here for the time being.

Reply:
im assuming you mean the opentelemetry config?